draft-ietf-rats-corim.md

```diff
@@ -2285,7 +2283,7 @@ See {{sec-authority}}.
 
 If the merged `measurement-values-map` contains duplicate codepoints and the
 measurement values are equivalent, then duplicate claims SHOULD be omitted.
-Equivalence typically means values MUST be binary identical.
+Equivalence means the values MUST be binary identical unless they are from a profile extension that defines its own equivalence relation.
```

Comment:
Move this to a top level Equivalence definition, and check, whether it is applicable in all sections!